front/src/common-app/components/table-player.component.tsx

```diff
@@ -24,6 +25,7 @@ export const TablePlayerComponent = React.forwardRef<HTMLTableElement, Props>(
 
     return (
       <div className={classes.container}>
+        <ScreenReaderSnackbarComponent />
```

Comment:
This is a global component, maybe we can use in `./src/app.tsx` and it will be available for all.

```
<ScreenReaderSnackbarProvider>
  <RouterComponent />
  <ScreenReaderSnackbarComponent />
</ScreenReaderSnackbarProvider>
```